cpp/mg_utility/mg_generate.hpp

```diff
@@ -10,20 +10,32 @@
 
 namespace mg_generate {
 
-/// Builds the graph from a given number of nodes and a list of edges.
+///
+/// @brief Builds the undirected graph from a given number of nodes and a list of edges.
 /// Nodes should be 0-indexed and each edge should be provided in both
```

Comment:
Edges should be in both directions to create an undirected graph?

Reply:
To create an undirected graph, one should specify edges that will act like undirected. For example, when we say `BuldGraph(2, {{0,1}, {1,0}}` we are creating two undirected edges, basically the same case as `BuldGraph(2, {{0,1}, {1,0}}`.